courses/fundamentals_of_ada/adv_120_advanced_privacy.rst

```diff
@@ -315,15 +327,23 @@ Child Units And Privacy
           X2 : T;
         end Root.Child;
 
-* Private child units are units that can be only made accessible to the private descendents of their parent
+* :dfn:`Private child` can
 
-  - Parent - visible in body
-  - Public siblings - visible in private section plus body
-  - Private siblings - visible in public and private sections plus body
+    + Used for "implementation details"
 
-  .. container:: columns
+---------------------------
+Importing a Private Child
+---------------------------
 
-    .. container:: column
+* A :ada:`private package` can view its **parent** :ada:`private` part
+* View on a private package is restricted
+
+   + To the *Private descendents of their parent*
```

Comment:
Move "restricted" to this bullet point - it doesn't apply to the others

Reply:
Yes, clarified the *view* a bit, otherwise it reads as "the view is visible from..."